src/components/Header/Header.tsx

```diff
@@ -0,0 +1,41 @@
+import { ChangeEvent, useState } from 'react';
+
+type Props = {
+  addTodo: (title: string) => void,
+};
+
+export const Header: React.FC<Props> = ({ addTodo }) => {
+  const [query, setQuery] = useState('');
+
+  const handleInput = (event: ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+  };
+
+  const handleSubmit = (event: ChangeEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!query) {
+      return;
+    }
+
+    addTodo(query);
+    setQuery('');
+  };
+
+  return (
+    <header className="header">
```

Comment:
Add a specific className or use a CSS module to avoid potential conflicts with existing CSS. Eg: Change the className 'header' to 'todo-header' or use CSS modules.